### PR TITLE
Architecture explanation

### PR DESCRIPTION
**Pull request by chinyakao**
add the README.md for description of MoonBear and steps of setup the project
add the ARCHITECTURE.md for architecture explanation

### CODE REVIEW

File: ARCHITECTURE.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+2. Outputs
+    - `TransformResponsesToCSV`: service to transform Launch.responses to CSV file
+    - `TransformDBSurveyToHTML`: service to create HTML strings: title & array of each survey page
+    - `TransformSheetsSurveyToHTML`: service to create HTML strings: title & array of each survey page
+3. Responses
+    - `StoreResponses`: send message(responses_hash) to queue
+4. Retrieve Surveys
+    - `GetSurveyFromDatabase`: return an entity of survey from database
+    - `GetSurveyFromSpreadsheet`: return an entity of survey from spreadsheet
+5. Surveys
```
Comment (soumyaray) [inline ARCHITECTURE.md:21]: Rename it to "Manage Surveys"

Comment (soumyaray) [reply ARCHITECTURE.md:21]: Move everything except start/close to a section called "Modify Surveys"

File: ARCHITECTURE.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    - (view::survey_export)
+6. `/privacy_policy` > (view::privacy_policy)
+
+### Infrastructure
+
+1. File store service (Google Drive)
+    - Stores spreadsheets??
+
+2. Spreadsheet service (Google Sheets)
+    - Fixed spreadsheet that is a Template for all new surveys
```
Comment (soumyaray) [inline ARCHITECTURE.md:61]: "Static" instead of "Fixed"

File: ARCHITECTURE.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        - *DETAILS: https://app.quickdatabasediagrams.com/#/d/ksihWW*
+        - `Account`: Login with google account
+        - `Survey`: a Account could create many survey
+        - `Launch`: if owner want to edit the survey, close the survey first and launch again after edited. 
+        - `Page`: different pages in a survey
+        - `Item`: item is the question in different page in different survey
+        - `Response`: each response is replied for each question(item) 
+    - TODO: Each deployed/previewed survey is stored in database??
+    - We store not only the responses to each question, we also store a copy of the survey structure for each respondent
+    - Prototype Design Pattern -- get video of it (Soumya)
```
Comment (soumyaray) [inline ARCHITECTURE.md:76]: video sent! maybe: "See video in team resources"

Comment (soumyaray) [reply ARCHITECTURE.md:76]: Let's make a shared Google Drive folder (Soumya)

File: README.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+# Survey MoonBear
+
+ Survey MoonBear provides a new way of survey services.
+
+## Link
+- Github: https://github.com/SurveyMoonBear/SurveyMoonbear_APP
+- Heroku: https://moonbear.herokuapp.com/
+
```
Comment (soumyaray) [inline README.md:8]: Let's make a resources section, with links to ARCHITECTURE.md etc.

Comment (soumyaray) [reply README.md:8]: Also link to DB graph

File: ARCHITECTURE.md (hunk 1 above)
Comment (chinyakao) [reply ARCHITECTURE.md]: We done~~

File: ARCHITECTURE.md (hunk 2 above)
Comment (chinyakao) [reply ARCHITECTURE.md]: Done

File: README.md (hunk 4 above)
Comment (chinyakao) [reply README.md:10]: Done

**Pull request merged**